StackStorm/st2-packages
# RFR - Bootstrap script fixes

## How to test?

* git clone https://github.com/lakshmi-kannan/st2-test-ground
* HOSTNAME=pkg-test-${flavor} vagrant up ${flavor}
* Notice comment before each flavor in Vagrantfile to see if you need to do any pre-requisites for that specific box. Run those commands.
* wget https://{location_to_bootstrap_script)
* chmod +x st2bootstrap-${flavor}.sh
* ./st2bootstrap-${flavor}.sh 
* Monkey test

## TODO

* [X] Install curl -  https://github.com/StackStorm/st2-packages/issues/183
* [X] Fix ubuntu script to run as non sudo - https://github.com/StackStorm/st2-packages/issues/184
* [x] Fix ubuntu script to be similar as EL7 -   https://github.com/StackStorm/st2-packages/issues/186
* [x] Install crudini in ubuntu - https://github.com/StackStorm/st2-packages/issues/187
* [X] EL6 bootstrap script 
    1. [x] WebUI can't be accessed outside virtualbox on centos66. Figure out if firewall is blocking.
* [x] Test EL script on an actual RHEL box
     1. [x] RHEL 7 -  (RHEL-7.2_HVM_GA-20151112-x86_64-1-Hourly2-GP2 (ami-2051294a))
     2. [ ] RHEL 6  - Packages don't work on RHEL6 because of missing libffi-devel. cc: @dennybaa 
* [x] Salvage docs for EL7 from https://gist.github.com/armab/2ab4d2ccc592db422439. https://github.com/StackStorm/st2docs/pull/75/files
* [x] Salvage docs for EL6 from https://gist.github.com/lakshmi-kannan/b885a6265e1a694ae0fd.
https://github.com/StackStorm/st2docs/pull/75/files
* [x] Chatops testing on EL6/EL7
   1. [x] No docker available for EL6 unless 3.10 kernel is installed by user. 
   2. [x] Tested docker story on EL7 and validated it works. 
* [x] Reboot box testing 
   1. [X] U14
   2. [x]  RHEL6 - st2 services don't come up. Manual st2ctl start needed.
   3. [x] RHEL7 - st2 services don't come up. Manual st2ctl start needed.
